gostream/source_stream_utils.go

```diff
@@ -83,6 +86,7 @@ func streamMediaSource[T, U any](
 			}
 			media, release, err := mediaStream.Next(ctx)
 			if err != nil {
```

Comment:
adding
```
error = errors.New("TESTING")
```
before this line is how I tested that below only show up when debug mode is on. Weirdly enough, it also breaks _just_ the live stream.